components/ai_chat/resources/untrusted_conversation_frame/components/assistant_task/use_extract_task_data.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import * as Mojom from '../../../common/mojom'
+
+// TODO(https://github.com/brave/brave-browser/issues/48535):
+// Should be Navigate and TODO tools.
+const importantToolNames = [Mojom.NAVIGATE_TOOL_NAME]
```

Comment:
Do you mind adding a comment explaining what makes a tool important & why we need to keep track of them?

Reply:
I have elaborated and made it clear it could change after collecting feedback:

```
// Important tools are ones worth always showing the latest version of
// to the user because they convey a longer-term status of the task compared
// to other tools. For example a navigation in a web page vs a click, or an
// up to date TODO list of items in the current task.
// TODO(https://github.com/brave/brave-browser/issues/48535):
// Should be Navigate and TODO tools.
// Note: Navigate could be removed if we display the current Url of a related
// content tab in the Task UI instead (since history navigation affects the
// "current" URL and not just the most recent navigation tool. This is all
// experimental.
```